charts/astronomer/values.yaml

```diff
@@ -318,6 +329,16 @@ configSyncer:
   # If not provided, will generate a random hour and minutes to spread cronjob workloads.
   schedule: ~
   securityContext: {}
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: false
+    # Annotations to add to the service account
+    annotations: {}
```

Comment:
if we are going to add annotation support then we need to extend that to our serviceAccount template as well currently its hardcoded for argo usage